google/cloud/bigtable/internal/table_admin.h

```diff
@@ -155,6 +156,55 @@ class TableAdmin {
 
   //@}
 
+  template <typename ResultType>
```

Comment:
I thought you were going to refactor this code, will that happen in a future PR?  If so, please add a TODO with a bug number to track that work.